api/pkg/app/app.go

```diff
@@ -370,9 +391,41 @@ func configFileURL() (string, error) {
 	return val, nil
 }
 
+// initGhe looks for Github Enterprise url from the environment variables
```

Comment:
nit: 
```suggestion
// InitGhe looks for Github Enterprise url from the environment variables
```

Reply:
`initGhe` is the name of the function hence kept it like that. Do I still need to change?

Reply:
You can keep it as it then 👍🏻